frappe/modules/import_file.py

```diff
@@ -100,6 +100,7 @@ def import_doc(docdict, force=False, data_import=False, pre_process=None,
 		ignore_version=None, reset_permissions=False):
 	frappe.flags.in_import = True
 	docdict["__islocal"] = 1
+	single_doctype = ['Print Settings'] #We are ignoring single doctype.
```

Comment:
Why ignore single doctypes? And why only Print Settings?

Reply:
@prssanna as per my understanding single doctype doesn't have any custom_fields and property setter, so single doctype having own fixtures separate file, hence added print Settings in list